conwayste/src/ui/layer.rs

```diff
@@ -92,7 +94,7 @@ pub struct Layering {
     // will have children nodes which are the nested elements
     // (think Buttons) of the widget.
     removed_node_ids:      HashSet<NodeId>, // Set of all node-ids that have been removed from the Tree
-    highest_z_order:       usize,           // Number of layers allocated in the system + 1
+    pub highest_z_order:   usize,           // Number of layers allocated in the system + 1
```

Comment:
This is kind of a red flag to me. What is mutating this?

Reply:
No mutation is going on. That is used in client.rs line 519 to determine if the `Run` state is at the lowest layer. I would have normally used a getter but you like to make these public so I oped for that., But it sounds like not in this case?